### PR TITLE
Feature: Add Movie

### PR DESCRIPTION
**Pull request by sarita95**
Added Movie class

### CODE REVIEW

File: MovieFestival/Movie.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		public Movie(int length, string genre, string title)
+		{
+			Length = length;
+			Genre = genre;
+			Title = title;
+		}
+
+		public string GetData()
+		{
+			string movie = $"{Title} - {Length} minutes - {Genre}";
```
Comment (marsicdev) [inline MovieFestival/Movie.cs:24]: Make sure that output is in format ex `Batman, 123min, DA` `Title, Length, Genre`

File: MovieFestival/Movie.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MovieFestival
+{
+	internal class Movie
```
Comment (marsicdev) [inline MovieFestival/Movie.cs:9]: Why is it internal class?

**Review by marsicdev — CHANGES_REQUESTED** [review]

File: MovieFestival/Movie.cs (hunk 1 above)
Comment (sarita95) [reply MovieFestival/Movie.cs]: Output format is changed to Title, Length, Genre

File: MovieFestival/Movie.cs (hunk 2 above)
Comment (sarita95) [reply MovieFestival/Movie.cs]: Class Accessibility Level changed to public

**Comment by marsicdev** [comment]
@sarita95 be more specific in commit message ex: `Change output to Name, length, genre` 

Hint: Use amend commit to change commit message

**Review by marsicdev — APPROVED** [review]
LGTM

**Pull request merged**